lib/Pusher.php

```diff
@@ -20,9 +20,9 @@
 		+ Phil Leggetter (phil@leggetter.co.uk)
 */
 
-class PusherException extends Exception
-{
-}
+require_once('PusherException.php');
+require_once('PusherHTTPException.php');
+require_once('TriggerResult.php');
```

Comment:
This will be the last reasonable update to this version of the library. I don't see any additional files being added to it in the future. It will only receive bug fixes.

We'll then move make the `new-lib` branch the master (probably a 4.0 release), do a new release and work from there following best practice.

So, it would be great if this version of the library could be consumed by those that don't use Composer.

> If you tell me what you prefer, I can even send you a PR with the change

That would be fantastic!